pkg/sources/errors_test.go

```diff
@@ -84,7 +86,7 @@ func TestScanErrorsAdd(t *testing.T) {
 				wg.Add(1)
 				go func() {
 					for j := 0; j < tc.wantErr/tc.concurrency; j++ {
-						se.Add(nil)
+						se.Add(testError)
```

Comment:
RE: https://github.com/trufflesecurity/trufflehog/pull/2608#discussion_r1535524068

My guess is that `nil` was allowed to make this test easier.